### PR TITLE
feat: improve map design

### LINKED ISSUE #513: Improve "How to find us" section
We could follow what was made on [JOIN website](https://join.di.uminho.pt).

### PR DESCRIPTION
**Pull request by joaodiaslobo**
Fix #513 , also moved the map to FAQs page since it makes way more sense than being on the Team page.

### CODE REVIEW

**Comment by joaodiaslobo** [comment]
[feat: improve map design](https://trello.com/c/neMsB87f/62-feat-improve-map-design)

File: layout/FAQs/components/FindUs/index.tsx [hunk 1]
```diff
@@ -15,10 +11,13 @@ export default function FindUs() {
         </h2>
 
         <p className="mb-8 font-iregular">
-          The SEI is free for participants and is organized by volunteers from
+          SEI is free for participants and is organized by volunteers from
           CeSIUM and from the university community.
         </p>
-
+        <p className="mb-8 font-iregular">
+          This years event will take place at Pedagocic Complex 2, Gualtar
```
Comment (ruilopesm) [inline layout/FAQs/components/FindUs/index.tsx:18]:
```suggestion
          This years event will take place at Pedagogic Complex 2, Gualtar
```

File: layout/FAQs/FAQs.tsx [hunk 2]
```diff
@@ -3,35 +3,38 @@ import { withoutAuth } from "@context/Auth";
...
-            to the most frequently asked questions here.
-          </p>
-        </div>
-        <div className="lg:w-1/3">
-          <div className="flex flex-col">
-            {questions.map((question, i) => (
-              <Question key={i} {...question} />
-            ))}
+    <>
+      <Navbar bgColor="primary" button={"quinary"} fgColor={"white"}>
```
Comment (ruilopesm) [inline layout/FAQs/FAQs.tsx:13]: Why are these curly braces needed?

**Review by ruilopesm — DISMISSED** [review]
Apart from that comment, great work 🙌🏼

File: layout/FAQs/FAQs.tsx (hunk 2 above)
Comment (joaodiaslobo) [reply layout/FAQs/FAQs.tsx:13]: No clue, they were already there 😅 I'll fix it anyway

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**